compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -123,7 +123,7 @@ constantDefinition
 globalVariableDefinition
     :   PUBLIC? LISTENER? typeName Identifier (ASSIGN expression)? SEMICOLON
     |   PUBLIC? FINAL (typeName | VAR) Identifier ASSIGN expression SEMICOLON
-    |   channelType Identifier SEMICOLON
+    |   channelType Identifier ASSIGN expression SEMICOLON
```

Comment:
@vinok88 Now we should be able to model channel as a normal type. Please create an issue for this. Let's fix this after the release.